File: topics/arrays/exercises/all-below.c
bool ????(const int data[], int sz, int max)
{
    int i;
    bool result = true;

    for (i = 0; i < sz; i++)
    {
        if (data[i] > max)
        {
          result = false;
          break;
        }
    }
    
    return result;
}

File: topics/arrays/exercises/AllBelow.pas
function ????(const data: array of Integer; max: Integer): Boolean;
var
    i: Integer;
begin
    result := true;

    for i := Low(data) to High(data) do
    begin
        if data[i] > max then
        begin
          result := false;
          break;
        end;
    end;
end;